packages/core/lib/commands/create/templates/Example.sol

```diff
@@ -2,6 +2,8 @@
 pragma solidity >=0.4.22 <0.9.0;
 
 contract Example {
-  constructor() public {
+  constructor() {
+    // Remove this and implement your contracts/Example.sol Smart Contract logic.
+    require(block.chainid != 1337, "You have to implement contracts/Example.sol's logic");
```

Comment:
oops, this isn't right. I am going to rethink this to support a better workflow.